serverless/admin_guide/serverless-ossm-setup.adoc

```diff
@@ -0,0 +1,40 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-setup"]
+= Integrating Service Mesh with OpenShift Serverless
+:context: serverless-ossm-setup
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Using {ProductShortName} with {ServerlessProductName} enables developers to configure additional networking and routing options.
+
+The {ServerlessOperatorName} provides Kourier as the default ingress for Knative. However, you can use {ProductShortName} with {ServerlessProductName} whether Kourier is enabled or not. The following procedures provide instructions for integrating {ProductShortName} with or without Kourier enabled.
+
+[IMPORTANT]
+====
+{ServerlessProductName} only supports the use of {ProductName} functionality that is explicitly documented in this guide, and does not support other undocumented features.
+====
+
+// without kourier
+[id="serverless-ossm-setup-native"]
+== Integrating {ProductShortName} with {ServerlessProductName} natively
```

Comment:
Should you use _without Kourier_ or when Kourier is not enabled_ to make it more parallel with the other heading? Then add _natively_ in the intro paragraph? 

__Integrating {ProductShortName} with {ServerlessProductName} if Kourier is not enabled, also known as _natively_..._

Reply:
I had this originally but our eng folks preferred this term, let me mull it over...

Reply:
I think I'll leave this as is for now - we can always change it later based on user feedback, but eventually the old Kourier docs will go away anyway and we can remove "natively" then